docs/de/developer-portal-page.md

```diff
@@ -225,6 +225,37 @@ Um den Transaction Status eines Payments abzufragen, fügen Sie Ihre Payment-Id
 | :------------- | :--------------------- | :--------- | :----------------- |
 | PSU-Successful | DE11760365688833114935 | finalised  | ACTC/ACSP          |
 
+
+### Confirmation of Funds
+
+Confirmation of Funds überprüft, ob der zu überweisende Betrag auf einem Konto vorhanden ist. Wenn der Kontostand ausreichend ist, wird "true" ausgegeben, anderenfalls wird "false" ausgegeben.
```

Comment:
The format of true and false is inconsistent to the english doc. Maybe we can use bold text? italic? or code format?

Reply:
done